src/main/markdown/release-notes.md

```diff
@@ -54,6 +55,62 @@ The GWT Release Notes
 **Note** - M1 = first milestone, RC1 = first release candidate
 
 * * *
+<a id="Release_Notes_Current"></a>
+## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+
+### Highlights
+
+- Java 8 syntax supported. 
+
+- [JsInterop new specification](https://docs.google.com/document/d/10fmlEYIHcyead_4R1S5wKGs1t2I7Fnp_PaNaa7XTEk0) implemented.
+
+- GSS is no longer considered as experimental and old /CssResource is now deprecated.
+
+### Deprecations
+
+- Old CssResource syntax is now deprecated.
+
```

Comment:
Classic dev mode deprecated.
(and instructions for switching back test cases temporarily)